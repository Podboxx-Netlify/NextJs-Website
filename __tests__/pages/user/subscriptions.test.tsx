import React from 'react'
import {shallow} from 'enzyme'
import Subscriptions from '../../../pages/user/subscriptions'
import {mockNextUseRouter} from "../../../__mocks__/routerMock";
import {act} from "react-dom/test-utils";

describe('Subscriptions', () => {
    let wrapper;
    jest.setTimeout(30000);
    beforeEach(() => {
        wrapper = shallow(<Subscriptions/>);
    })

    it('should render without crashing', () => {
        expect(wrapper).toMatchSnapshot();
    })

    // it('Page should be loading for 5sec then redirect to login', async () => {
    //     // jest.useFakeTimers();
    //
    //     await page.goto('http://localhost:5000/user/login')
    //     await expect(page).toMatch('login')
    //
    //     // act(() => {
    //     //     jest.advanceTimersByTime(6000)
    //     // })
    //
    //     // expect(wrapper.contains(<div className="cover-spin" id="cover-spin"/>)).toEqual(false);
    //     // expect(setTimeout).toHaveBeenCalledTimes(1)
    // })
})