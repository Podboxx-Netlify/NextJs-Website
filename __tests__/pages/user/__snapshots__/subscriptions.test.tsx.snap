// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscriptions should render without crashing 1`] = `
<div
  className="cover-spin"
  id="cover-spin"
/>
`;
